play/PlayerRules/CharacterCreation/02_Classes.txt

```diff
@@ -280,13 +289,15 @@ Expend 10 nanites to paint a target. All damage dealt by allies to this target
 is increased by 20, and any perception checks are 50% easier to pass. Use 
 twice per encounter. Level number of uses per day. Lasts for the day.
 
-//needs work
-"Paladin" - You are a righteous warrier, vetted and trained by one of the 
+"Paladin" - You are a righteous warrior, vetted and trained by one of the 
+
 powerful corporations or some other technologically advanced entity. You seek 
 to uphold the principals which define the group you''ve sworn allegience to. 
 Through this devotion you have been granted access to high tech in order to 
 pursue your goals and missions.
-Passives: Terms and Conditions clause compliance, access to Paladin Spell tree
+Passives: You gain 45 total nanites and each level you gain an additional 5 
+maximum nanites for each level after your first.
+Terms and Conditions clause compliance, access to Paladin Spell tree
```

Comment:
"Terms and Conditions clause compliance"??? is this a special power? I am not aware of a clause compliance

Reply:
Paladin code of conduct. I'll add an issue